replications/internal/queue_management_test.go

```diff
@@ -21,8 +19,6 @@ var (
 )
 
 func TestCreateNewQueueDirExists(t *testing.T) {
-	t.Parallel()
```

Comment:
Why drop all the `Parallel()` calls here?

Reply:
I was having trouble with races in tests, which made me realize that running these tests in parallel could cause issues where one replication ID queue has multiple goroutines acting on it, and multiple durable queue managers running at once. I might be wrong on this, but removing these parallel calls removed some race issues I was having that were only within the test and not the actual logic

Reply:
I would be hesitant to assume the race conditions are only in the tests. There shouldn't be any shared state between instances of the queue manager between tests, so even when tests reuse replication IDs I'd expect it to be safe to have multiple concurrent instances running. I'd never expect to get into a state where multiple goroutines end up pointing at the same queue/queue manager.